/*
 * ConfigInc.h
 *
 *  Created on: 2016-8-22
 *      Author: Ralf
 */

#ifndef CONFIGINC_H_
#define CONFIGINC_H_

#include "Demo.pb.h"
#include "BagConfig.pb.h"
#include "OlEquipmentShop.pb.h"
#include "OlEquipment.pb.h"
#include "Building.pb.h"
#include "PassSystemConfig.pb.h"
#include "Tavern.pb.h"
#include "Hero.pb.h"
#include "HeroAttack.pb.h"
#include "UserConfig.pb.h"
#include "ResourceConfig.pb.h"
#include "MissionConfig.pb.h"
#include "NPC.pb.h"
#include "ScienceConfig.pb.h"
#include "EquipSmeltConfig.pb.h"
#include "VipGiftConfig.pb.h"
#include "WeaponConfig.pb.h"
#include "MarketConfig.pb.h"
#include "WorldRewardConfig.pb.h"
#include "CommonGiftConfig.pb.h"
#include "ExploitBoxConfig.pb.h"
#include "RewardConfig.pb.h"
#include "GovernAffair.pb.h"
#include "VIPConfig.pb.h"
#include "EquipSuitConfig.pb.h"
#include "SpecialEquipConfig.pb.h"
#include "ShopConfig.pb.h"
#include "KillRankConfig.pb.h"
#include "OfficialConfig.pb.h"
#include "FogGateConfig.pb.h"
#include "MineConfig.pb.h"
#include "ActivityConfig.pb.h"
#include "BotConfig.pb.h"
#include "DailyMissionConfig.pb.h"
#include "ComponentConfig.pb.h"
#include "AchievementConfig.pb.h"
#include "ShakeDiceConfig.pb.h"
#include "LittleChargeConfig.pb.h"
#include "ResourceCopyConfig.pb.h"
#include "EnlivenKillConfig.pb.h"
#include "OperaBattleConfig.pb.h"
#include "ModelTrialConfig.pb.h"
#include "CrossBusinessConfig.pb.h"
#include "AttackHarborConfig.pb.h"
#include "WorldEliteConfig.pb.h"
#include "SellComponentConfig.pb.h"

#endif /* CONFIGINC_H_ */
